server/http.go

```diff
@@ -29,11 +32,213 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		p.handleWebhook(w, r)
 	case "/api/v1/meetings":
 		p.handleStartMeeting(w, r)
+	case "/oauth2/connect":
+		p.connectUserToZoom(w, r)
+	case "/external/oauth2/connect":
+		p.connectExternalUserToZoom(w, r)
+	case "/oauth2/complete":
+		p.completeUserOAuthToZoom(w, r)
 	default:
 		http.NotFound(w, r)
 	}
 }
 
+func (p *Plugin) connectUserToZoom(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get("Mattermost-User-ID")
+	if userID == "" {
+		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		return
+	}
+
+	channelID := r.URL.Query().Get("channelID")
+
+	if channelID == "" {
+		http.Error(w, "Not authorized", http.StatusUnauthorized)
```

Comment:
Is `Not authorized` the correct error message or is this from a channel not existing?  `Not authorized` is checked above